fastly.toml

```diff
@@ -3,3 +3,15 @@ name = "Default starter for Rust"
 description = "Get to know the Fastly Compute@Edge environment with a basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 authors = ["<oss@fastly.com>"]
 language = "rust"
+
+[setup]
```

Comment:
What do you think about prefacing this section with a comment mentioning that this is purely for the CLI and initializing a project? We've seen a couple reports now of people trying to add a `[setup]` section to their project, and being confused that it hadn't matched their expectations.

Reply:
> We've seen a couple reports now of people trying to add a [setup] section to their project, and being confused that it hadn't matched their expectations.

In that case, definitely! sounds like a good idea